src/geo/ui/search.js

```diff
@@ -245,7 +245,7 @@ var Search = View.extend({
   clean: function() {
     this._unbindEvents();
     this._destroySearchPin();
-    this.elder('clean');
+    Search.__super__.clean.call(this);
```

Comment:
:+1: I should have looked into this. Two more reasons to not rely on `__super__`:

 - https://github.com/jashkenas/backbone/pull/3539#issuecomment-82384843
 - https://github.com/jashkenas/backbone/pull/787#issuecomment-3143358

I will take of it.